packages/amplify-e2e-core/src/utils/retrier.ts

```diff
@@ -4,7 +4,7 @@ import { sleep } from './sleep';
 const defaultSettings: RetrySettings = {
   times: Infinity,
   delayMS: 1000 * 10, // 10 seconds
-  timeoutMS: 1000 * 60 * 20, // 20 minutes
+  timeoutMS: 1000 * 60 * 25, // 25 minutes
```

Comment:
What did you encounter that caused the bump from 20 to 25 minutes on the timeout?

I would guess this means our e2e tests slowed down by ~25%, which might be enough to be worth an investigation. If you have already done the investigation, whats driving the increase and can we do better?

Reply:
There is a performance degradation with newer versions of Jest when run with node < 20 (https://github.com/jestjs/jest/issues/11956). We currently support Node 18. Node 18 will be EOL in 8 months, so we should look into upgrading to Node 20.